iree/compiler/Dialect/Stream/Conversion/PatternUtils.cpp

```diff
@@ -37,9 +37,10 @@ ConvertedTensor consumeTensorOperand(Location loc, Value operand,
         castOp.getOperand(1),
     };
   }
-  llvm_unreachable(
-      "unexpected operand; should have either been converted from tensor or "
-      "not");
+  assert(false &&
+         "unexpected operand; should have either been converted from tensor or "
+         "not");
```

Comment:
wat. `P ∨ ~P = true` so I don't understand what this message is saying. Maybe we can fix it while we're here?

Reply:
done.